### DISCUSSION TITLE
Keeping materials and colors from one printer to another.

### DISCUSSION THREAD

**Discussion by JonnyHardCash** [OrcaSlicer/OrcaSlicer#6875] [Q&A]
Hello everyone, maybe I am the ignorant here, but why does orca slicer not keep all the materials when switching from one printer to another? Why do I have to redo the entire painting session when I change from one printer to another? Is it me getting some option wrong or is this absurdity embedded in the software?

**Comment by CuredPrusa** [comment]
Having the same question, I find yours here.
Why is filament list depending on printer type without a simple way to share same filament among more printers? In Prusa you klick and interchange those three settings (Print Setting /Filament /Printer) as you like, without so much unnecessary steps as required in Orca.
If this is not a bug but deliberate software "feature", I'd like to hear a reasonable explanation because I can not find any. It's the same as if my Skoda Octavia can run on Skoda Octavia tyers only. Such a powerful software with such unbelievably awkward workflow for basic simple setting.

**Comment by dexy31337** [comment]
Any news on this one?
